R/TealData.R

```diff
@@ -193,6 +193,28 @@ TealData <- R6::R6Class( # nolint
       private$join_keys$get_parents()
     },
 
+    #' @description
+    #' returns the `tdata` object.
+    #'
+    #' @return (`tdata`) object of the datasets.
+    get_tdata = function() {
```

Comment:
I don't understand the need for this method. And I don't see it used anywhere throughout Teal.

Reply:
So this is used in teal 
![image](https://user-images.githubusercontent.com/15201933/222093201-2231c12e-52bb-4e0c-b630-2c4a45fdfdaf.png)

Reply:
Ah OK, missed that.